### ISSUE TITLE
Tokens: glacier-blue-40 contrast 2.99:1 is not sufficient

### ISSUE THREAD

**Issue by lorilundy711** [ithaka/pharos#602]
**The problem**
The contrast of glacier-blue-40 is 2.99:1 which does not meet the WCAG focus-appearance criteria https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance

**The solution**
Keep the same color token name for glacier-blue-40, but change the hex code to #0B9FBC (lightness 39 instead for 40 in the HSL) which increases the contrast to 3.13:1.

**Comment by sirrah-tam** [comment]
Quick note: 2.4.13 Focus Appearance is slated to be an AAA Success Criteria (SC) as part of WCAG 2.2 which is said to be published in Sept 2023. This particular issue could fall under the 1.4.11 Non-text Contrast (AA) SC where the active focus state doesn't not meet that 3:1 threshold.

**Comment by daneah** [comment]
@sirrah-tam I'm having trouble parsing whether your note requires additional/different action than what @lorilundy711 describes.

**Comment by sirrah-tam** [comment]
@daneah - Oop, good call out. My note was one of clarification since the WCAG Success Criteria referenced in the initial issue isn't technically in the wild yet and that its an AAA criteria which we can strive for but aren't required to conform to. A case could be made using the other WCAG SC referenced for making this change, though its one that some could argue against.

Either way, the solution proposed in the initial issue is the recommended path forward IMO!

**Comment by daneah** [comment]
@sirrah-tam Awesome, thanks for clarifying!
